qiskit/quantum_info/operators/utils.py

```diff
@@ -0,0 +1,96 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Quantum information utility functions for operators.
+"""
+
+from qiskit.quantum_info.operators.linear_op import LinearOp
+
+
+def commutator(a: LinearOp, b: LinearOp) -> LinearOp:
+    r"""
+    Compute commutator of a and b.
+
+    .. math::
+
+        ab - ba.
+
+    Args:
+        a: Operator a.
+        b: Operator b.
+    Returns:
+        The commutator
+    """
+    return a @ b - b @ a
+
+
+def anti_commutator(a: LinearOp, b: LinearOp) -> LinearOp:
+    r"""
+    Compute anti-commutator of a and b.
+
+    .. math::
+
+        ab + ba.
+
+    Args:
+        a: Operator a.
+        b: Operator b.
+    Returns:
+        The anti-commutator
+    """
+    return a @ b + b @ a
+
+
+def double_commutator(a: LinearOp, b: LinearOp, c: LinearOp, *, sign: bool = False) -> LinearOp:
+    r"""
+    Compute symmetric double commutator of a, b and c.
+    See McWeeny chapter 13.6 Equation of motion methods (page 479)
```

Comment:
With a single header line followed by an empty line this renders nicely in the docs 🙂 
```suggestion
    r"""Compute symmetric double commutator of a, b and c.

    See McWeeny chapter 13.6 Equation of motion methods (page 479)
```